src/masonite/drivers/mail/BaseMailDriver.py

```diff
@@ -39,7 +52,10 @@ def to(self, user_email):
         if callable(user_email):
             user_email = user_email.email
 
-        self.to_address = user_email
+        if isinstance(user_email, (list, tuple)):
+            self.to_addresses = user_email
```

Comment:
Should this be `+=` maybe? it would make it able to be called several times

Reply:
So how would you reset?